### PR TITLE
Export the InputTooLongException

### PR DESCRIPTION
**Pull request by lukef**
It's not possible to check for `InputTooLongException` because it isn't exported.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .dart_tool
 .pub
 pubspec.lock
+.idea
```
Comment (kevmoo) [inline .gitignore]: Please revert this change. These ignores should be in `~/.gitignore_global`

Comment (lukef) [reply .gitignore]: Done

**Comment by lukef** [comment]
I decided to go with `1.2.0` as it's not exactly a patch, but let me know if you want me to swap that.
